src/services/utilities.ts

```diff
@@ -469,6 +469,33 @@ namespace ts {
         }
     }
 
+    /**
+     * Get the corresponding JSDocTag node if the position is in a jsDoc commet
+     */
+    export function getJsDocTagAtPosition(sourceFile: SourceFile, position: number): JSDocTag {
+        let node = ts.getTokenAtPosition(sourceFile, position);
+        let jsDocComment: JSDocComment;
+        while (true) {
+            if (node === sourceFile ||
+                node.kind === SyntaxKind.VariableStatement ||
+                node.kind === SyntaxKind.FunctionDeclaration ||
+                node.kind === SyntaxKind.Parameter ||
+                node.jsDocComment) {
+                jsDocComment = node.jsDocComment;

```

Comment:
@mhegazy For now only these three kinds of node will be attached with jsDoc comments, as shown in [parser code](https://github.com/Microsoft/TypeScript/blob/master/src/compiler/parser.ts#L606)